### ISSUE TITLE
Autocomplete doesn't work

### ISSUE THREAD

**Issue by Abdulrahman-Tayara** [withfig/fig#2535]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 12.2.1 (21D62)

### Expected behaviour

I'm using Fig for 8 months and the autocomplete was working fine. 

### Actual behaviour

But now when I write any command on the terminal I didn't get any suggestions/autocomplete from fig. 
I ran `fig doctor` and tried to restart the application but everything failed.

`fig doctor` output:
```
✔ Everything looks good!

  Fig still not working? Run fig issue to let us know!
  Or, email us at hello@fig.io!
```

I attached `Autocomplete Devtools` console logs.

<img width="997" alt="image" src="https://github.com/withfig/fig/assets/43582467/79d20c06-1550-45e3-a38e-d7e66c4a9193">

### Environment

```yaml
fig-details:
  - 2.16.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1
  - cores: 8
  - mem: 8.00 GB
os-info:
  - macOS 12.2.1 (21D62)
environment:
  - shell: /bin/zsh
  - terminal: terminal
  - cwd: /Users/abdulrahman
  - exe-path: /Users/abdulrahman/.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: 9140b197-c182-4332-859c-67ace4a5610e
    - FIG_SET_PARENT_CHECK: 1
    - FIG_TERM: 2.16.0
    - PATH: /Users/abdulrahman/miniforge3/bin:/Users/abdulrahman/miniforge3/condabin:/usr/local/opt/ruby/bin:/usr/local/lib/ruby/gems/3.1.0/bin:/opt/homebrew/opt/ruby/bin:/Users/abdulrahman/Downloads/google-cloud-sdk/bin:/opt/homebrew/opt/node@16/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Users/abdulrahman/.cargo/bin:/Users/abdulrahman/.fig/bin:/Users/abdulrahman/.local/bin:/Users/abdulrahman/Library/Application Support/JetBrains/Toolbox/scripts
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - __CFBundleIdentifier: com.apple.Terminal
    - FIG_PID: 4080
    - FIG_SET_PARENT: 9140b197-c182-4332-859c-67ace4a5610e
```

**Comment by KeyToLove** [comment]
same issue here, my system info:
```
  System:
    OS: macOS 12.2
    CPU: (10) arm64 Apple M1 Pro
    Memory: 221.27 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
```
any solutions ?

**Comment by zhkjim** [comment]
I got the same problem after I update my MacOS version to 13.3

**Comment by electroheadfx** [comment]
Same issue:
```
fig-details:
  - 2.16.0-beta.3
hardware-info:
  - model:
  - model-id:
  - chip-id: Apple M1 Pro
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 12.6.3 (21G419)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/lmarques/Dev/railwai-nx-next13
  - exe-path: /Users/lmarques/.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: 51e57934-e00c-4631-ad8c-1e35ec5226e5
    - FIG_SET_PARENT_CHECK: 1
    - FIG_TERM: 2.16.0-beta.3
    - PATH: /Users/lmarques/Library/pnpm:/Users/lmarques/.nvm/versions/node/v16.17.1/bin:/Library/Scripts/google-cloud-sdk/bin:/Applications/Sublime Text.app/Contents/SharedSupport/bin:/opt/homebrew/bin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/lmarques/.fig/bin:/Users/lmarques/.local/bin:/Library/Scripts/efx:/Users/lmarques/Library/Python/3.9/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - __CFBundleIdentifier: com.googlecode.iterm2
    - FIG_PID: 26058
    - FIG_SET_PARENT: 51e57934-e00c-4631-ad8c-1e35ec5226e5
    ```

**Comment by duyhunter1001** [comment]
the same this issue to fixed at here https://github.com/withfig/fig/issues/2522

**Comment by Abdulrahman-Tayara** [comment]
> the same this issue to fixed at here #2522

@duyhunter1001  - Thank you, it works now. 

I will close this issue.
